db/migrate/20150217202928_create_keywords.rb

```diff
@@ -0,0 +1,14 @@
+class CreateKeywords < ActiveRecord::Migration
+  def change
+    unless ActiveRecord::Base.connection.table_exists? 'keywords'
+      create_table :keywords do |t|
+        t.string :keyword, limit: 1000
```

Comment:
maybe this field name should be changed to :name or :title  because the current field name causes different situation when it's not clear enough what is calling in the code like: @keyword.keyword. More readable would be @keyword.name